kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/css/denali.css

```diff
@@ -1041,13 +1041,6 @@ Packages panel dropzone START
     color:  white;
 }
 
-.packages-panel {
```

Comment:
Please change header to: "* Copyright (c) 2011, 2020 Eurotech and/or its affiliates and others"